src/main/java/core/basesyntax/service/impl/FileWriteServiceImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileWriteService;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Map;
+import java.util.Set;
+
+public class FileWriteServiceImpl implements FileWriteService {
+    @Override
+    public boolean writeFile(Map<String, Integer> fruits) {
+
+        Set<Map.Entry<String, Integer>> entries = fruits.entrySet();
+        try (FileWriter fileWriter = new FileWriter("output.txt")) {
```

Comment:
It's hard code of output file path. Let's recive it as program arguments like you doing with input file.